tests/integration/test_schema.py

```diff
@@ -1863,7 +1863,13 @@ async def test_invalid_namespace(registry_async_client: Client) -> None:
     subject = create_subject_name_factory("test_invalid_namespace")()
     schema = {"type": "record", "name": "foo", "namespace": "foo-bar-baz", "fields": []}
     res = await registry_async_client.post(f"subjects/{subject}/versions", json={"schema": jsonlib.dumps(schema)})
-    assert res.ok, res.json()
+    assert res.status == 422, res.json()
+    json_res = res.json()
+    assert json_res["error_code"] == 44201, json_res
+    assert (
+        json_res["message"] == "Invalid AVRO schema. "
+        "Error: foo-bar-baz is not a valid Avro name because it does not match the pattern (?:^|\\.)[A-Za-z_][A-Za-z0-9_]*$"
+    ), json_res
```

Comment:
```suggestion
    expected_message = (
        "Invalid AVRO schema. Error: foo-bar-baz is not a valid Avro name because it does not match the pattern (?:^|\\.)[A-Za-z_][A-Za-z0-9_]*$"
    )
    assert json_res["message"] == expected_message, json_res
```

Reply:
Done.